doc/patchutils.xml

```diff
@@ -1449,7 +1455,368 @@ done)]]></screen></para>
+      </refnamediv>
+
+      <refsynopsisdiv>
+	<cmdsynopsis>
+	  <command>patchview</command>
+	  <group choice="opt">
+	    <arg>-n</arg>
+	    <arg>--line-number</arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-p <replaceable>n</replaceable></arg>
+	    <arg>--strip-match=<replaceable>n</replaceable></arg>
+	  </group>
+	  <arg choice="opt">--strip=<replaceable>n</replaceable></arg>
+	  <arg choice="opt">--addprefix=<replaceable>PREFIX</replaceable></arg>
+	  <group choice="opt">
+	    <arg>-s</arg>
+	    <arg>--status</arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-E</arg>
+	    <arg>--empty-files-as-removed</arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-i <replaceable>PATTERN</replaceable></arg>
+	    <arg>--include=<replaceable>PATTERN</replaceable></arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-x <replaceable>PATTERN</replaceable></arg>
+	    <arg>--exclude=<replaceable>PATTERN</replaceable></arg>
+	  </group>
+	  <group>
+	    <arg>-z</arg>
+	    <arg>--decompress</arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-# <replaceable>RANGE</replaceable></arg>
+	    <arg>--hunks=<replaceable>RANGE</replaceable></arg>
+	  </group>
+	  <arg choice="opt">--lines=<replaceable>RANGE</replaceable></arg>
+	  <group choice="opt">
+	    <arg>-F<replaceable>RANGE</replaceable></arg>
+	    <arg>--files=<replaceable>RANGE</replaceable></arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-H</arg>
+	    <arg>--with-filename</arg>
+	  </group>
+	  <group choice="opt">
+	    <arg>-h</arg>
+	    <arg>--no-filename</arg>
+	  </group>
+	  <group choice="opt" rep="repeat">
+	    <arg>-v</arg>
+	    <arg>--verbose</arg>
+	  </group>
+	  <arg choice="opt" rep="repeat"><replaceable>file</replaceable></arg>
+	</cmdsynopsis>
+
+	<cmdsynopsis>
+	  <command>patchview</command>
+	  <group choice="req">
+	    <arg>--help</arg>
+	    <arg>--version</arg>
+	    <arg>--filter ...</arg>
+	    <arg>--grep ...</arg>
+	  </group>
+	</cmdsynopsis>
+      </refsynopsisdiv>
+
+      <refsect1>
+	<title>Description</title>
+
+	<para>So what is patchview ? is a wrapper of filterdiff which aim to be used with numbered files. patchview without options or args do a: lsdiff --number. patchview with options or args (for example "-F2-") do a filterdiff with the same options and args </para>
+
+	<para>And we have two scripts for working with git and other two for working with svn, for git we got gitdiff and gitdiffview,
+for svn we got svndiff and svndiffview.
+
+    $ svndiff or gitdiff (without args) will give the list of files modified
+
+    $ svndiff -F1 or gitdiff -F1 (with args) will show the patch of file #1
+
+    $ svndiffview or gitdiffview (without args) pipe all patch with filerdiff to vim - -R (in ready only mode, easy to
+    quit) , and we will see colored and complete patch,
+
+    $ svndiffview -F2 or gitdiffview -F2 (with args) , it will pipe patch of file #2 to vim - -R
```

Comment:
Can we use some of the README corrections here?

Reply:
I haven't sent yet , I will finish it in a couple of hours